Header header
string asifStatus
float64 TTS
float64 BTorthoBS
float64 hBackupEnd
float64 hSafetyNow
float64[] hDebug
int32 indexDebug
float64[4] xBackupEnd
float64 relax1
float64 relax2
float64 filterTimerUs
float64[] DhDebug
float64[] Lfh_diff
float64[] Lgh_diff

float64[] h_full
float64[] traj_full
float64[] lfh_full
float64[] lgh_full
float64[] Dh_full

float64[] f_full
float64[] g_full